### ISSUE TITLE
Same bounding box for all images

### ISSUE THREAD

**Issue by Fenil3510** [experiencor/keras-yolo2#403]
I'm training using the notebook Blood_Cell_Detection.ipynb on custom dataset.
I'm also not using any pretrained weights. So after 7-8 epochs(initial 3 were warmup), average recall is around 0.75 and the train loss and val loss are continuously decreasing , but looks like its predicting the same bounding box for all the images.
These are some of the train set images.
It would be great if someone help me with this
Thanks in advance

![whatsapp image 2019-02-05 at 12 46 13 am](https://user-images.githubusercontent.com/20499290/52251541-92808080-2923-11e9-9414-30bdf8d6d7b7.jpeg)
![whatsapp image 2019-02-05 at 12 46 10 am](https://user-images.githubusercontent.com/20499290/52251542-944a4400-2923-11e9-99ec-61224c0b355b.jpeg)
![whatsapp image 2019-02-05 at 12 46 11 am](https://user-images.githubusercontent.com/20499290/52251545-96140780-2923-11e9-9047-5bee85a06d3d.jpeg)
